src/templates/app/user/security/self_exclusion.jsx

```diff
@@ -32,9 +32,9 @@ const SelfExclusion = () => (
                 <FormRow type='text' id='max_balance' label={it.L('Maximum account cash balance')} attributes={{ maxLength: 20 }} className='prepend_currency' hint={it.L('Once this limit is reached, you may no longer deposit.')} />
 
                 <div data-show='iom' className='form-row no-padding'>
-                    <FormRow type='text' id='max_deposit' label={it.L('Maximum deposit amount limit')} attributes={{ maxLength: 20 }} className='prepend_currency' hint={it.L('Once this limit is reached, you may no longer deposit.')} />
+                    <FormRow type='text' id='max_deposit' label={it.L('Maximum deposit limit')} attributes={{ maxLength: 20 }} className='prepend_currency' hint={it.L('Once this limit is reached, you may no longer deposit.')} />
 
-                    <FormRow type='text' id='max_deposit_end_date' label={it.L('Limit applies until')} attributes={{ maxLength: 15, autoComplete: 'off', readOnly: 'readonly' }} className='clearable' hint={it.L('Please enter date in the format DD MMM, YYYY.')} />
+                    <FormRow type='text' id='max_deposit_end_date' label={it.L('Maximum deposit limit expiry')} attributes={{ maxLength: 15, autoComplete: 'off', readOnly: 'readonly' }} className='clearable' hint={it.L('Please enter date in the format DD MMM, YYYY.')} />
```

Comment:
I thought this would be `Maximum deposit amount limit applies until to Maximum deposit limit until`?

Reply:
@ahkuma no, check the comments on the card